.github/actions/install-airbyte-ci/action.yml

```diff
@@ -10,20 +10,28 @@ inputs:
     description: "Path to airbyte-ci source"
     required: false
     default: airbyte-ci/connectors/pipelines
+  is_fork:
+    description: "Whether the PR is from a fork"
+    required: false
+    default: "false"
 runs:
   using: "composite"
   steps:
     - name: Get changed files
       uses: tj-actions/changed-files@v39
```

Comment:
I have a feeling that this is outdated, but I don't remember. I think the new one is v43?